src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -182,8 +184,14 @@ export class InstructorSessionsPageComponent extends InstructorSessionBasePageCo
    * Sets default values for the session edit form.
    */
   initDefaultValuesForSessionEditForm(): void {
-    // select the first course by default
-    if (this.courseCandidates.length !== 0) {
+    // if specified in the query and is valid, select that course
+    // otherwise, select the first course by default
+    const course: any = this.getCourseFromList(this.courseId);
```

Comment:
I think you can inline the method `getCourseFromList`. It's more clear.

`any` -> `Course | undefined`

Reply:
Makes sense. Updated.